src/common/guac_surface.c

```diff
@@ -437,8 +437,8 @@ static void __guac_common_surface_put(unsigned char* src_buffer, int src_stride,
 
     int x, y;
 
-    int min_x = rect->width - 1;
```

Comment:
What is the logic behind this change?

If `min_x` and `min_y` are truly to be the minimum possible values of X and Y within the service, as their name and usage imply, then they are correctly calculated as `rect->width - 1` and `rect->height - 1` respectively.

It further looks like this would interfere with the width and height calculations toward the end (around line 488).

I guess I'm just not seeing how this wasn't working before, and how this fixes that problem. Is there some comparison happening later that is not using `min_x` and `min_y` correctly?